h8mail/utils/classes.py

```diff
@@ -319,7 +315,7 @@ def get_emailrepio(self, api_key=""):
                         self.data.append(
                             (
                                 "EMAILREP_LEAKS",
-                                "{} leaked credential".format(data["references"]),
+                                f"{data['references']} leaked credential",
```

Comment:
I'm not sure it's a best way but it looks well

Reply:
I will manually test this one for safety

Reply:
Looks good, I need to add error check for code 400 later